README.md

```diff
@@ -22,23 +22,23 @@ kubectl apply -f metric_anomaly_detector.yaml
 Make sure to update the metric_anomaly_detector.yaml file to point to the correct image path!
 
 #### Import Grafana Dashboard
-1. Navigate to Grafana and login. For Rancher monitoring user, the username/password is `admin/prom-operator`.
-2. [Add Elasticsearch as datasource](https://grafana.com/docs/grafana/latest/datasources/add-a-data-source/#add-a-data-source), and fillin these fields with following values:
+1. Navigate to Grafana and login. For Rancher monitoring user, the username/password is `admin/prom-operator`, otherwise it's likely to be `admin/admin`
```

Comment:
```suggestion
1. Navigate to Grafana and log in. For a Rancher Monitoring user, the default username/password is `admin/prom-operator`. Otherwise, it is likely to be `admin/admin`.
```